etcdutils.go

```diff
@@ -77,7 +77,7 @@ users:
 `
 	t := template.Must(template.New("config").Parse(configTemplate))
 	if err = t.Execute(os.Stdout, params); err != nil {
-		log.Fatal(err)
+		fmt.Fprintf(os.Stderr, "Error executing template %v\n", err)
```

Comment:
the `log.Fatal()` will `os.Exit(1)`, are you aware that `fmt.Fprintf` does not do that?

Reply:
That's exactly why I removed log.Fatal and added error, and return below the line.